tasks/Bullard Houses.md

```diff
@@ -0,0 +1,48 @@
+# Summary
+The Bullard House experiment is designed to test negotiations and agreement-bias.  In this experiment the two roles are designed to be at odds with their interests, which creates a negative bargaining zone.
+
+# References
+Cohen, T., G. Leonardelli, and L. Thompson. "The agreement bias in negotiation: Teams facilitate impasse." 23rd annual conference of the International Association for Conflict Management (IACM), Boston, US. 2010.
+
+Link: https://d1wqtxts1xzle7.cloudfront.net/51359748/The_Agreement_Bias_in_Negotiation_Teams_20170115-19001-1910usu-with-cover-page-v2.pdf?Expires=1630165799&Signature=Leq9jWOmOoglLUt7pY1HrlvFlv47s314LbCYqHmpGIlhMLYkvCmDk5O8qIhvTw8LR75oyeaMHvX34frVvw6KbYkMGLaBaIj3hawbz-VMEwtskNCS-5YwB30ge9Dte~PYf20bSmoLeXAa2uIw1rhsrdO7FOLU9dh0yVbXlpnYpHB6T4o-zWFiZJompKkdHgIjjC6pawHzMms9DFBmPxZJwgaAvEuUAbKgE3uZBUGpJCeM-is8TKiiX-91ipoWwosfZxArMaP2AM0UwGmWiiyEH7JwCgF1YrByhGQdCFhObUm6wGPabxyBvYfvZc2H9F~vDwPpZcFMPSS9ALOuxpHDuA__&Key-Pair-Id=APKAJLOHF5GGSLRBV4ZA
+
+
+# Stimuli
+## The visual components
+N/A
+
+
+## Materials for alternative versions of the experiment 
+Alternate versions of the experiment allow you to introduce teams.  These experiments allow you to test the effects of group negotiations.  Different formats tested in, The agreement bias in negotiation: Teams facilitate impasse were, one-on-one, two-on-two and three-on-three.
+
+
+# Procedure
+Groups of participants were selected from MBA students.  (At this point you can break the groups into teams, or use one-on-one negotiations. Teams are assigned randomly.)  Participants are then given their role materials one week prior to negotiations.
+
+On negotiation day, both parties are to attempt to reach deal while acting in their roles.  
+
+After negotiations cease participants should report whether they have reached an agreement or an impasse.  These should be coded, i.e. agreement(0) and impasse(1).
+
+## Steps
+1. (Optional) break participants into groups
+2. Disperse role materials to participants one week prior to negotiations
+3. Allow participants to negotiate
+4. After negotiations conclude, ask participants to report if they reached a deal. (i.e. agreement(0) and impasse(1))
+
+
+## Roles 
+Seller, this role is to act as an agent on behalf of a retail company representing the Bullard Family, who is selling their property. 
+Buyer, this role is to act as an agent representing a blind trust company operated by a hotel group.
+
+## Instructions
+Seller, this role is instructed to only sell the property to a known buyer who fully discloses their planned use of the property, while also seeking to get a fair deal for the family.
```

Comment:
I think the actual negotiation needs to be purchased for the instructions. 

Here's a overview from the site: 
Downtown Realty, Inc. owns the historic Bullard Houses, a set of 51 attached brownstones in the city of Gotham. The Houses, occupied for decades by the city's wealthy elite, have fallen into disrepair and are currently occupied only by a few low-income families. Downtown Realty has been prevented from demolishing the Houses by the Gothic Landmark Commission. Consequently, Downtown is eager to sell the property, and has several offers on the table. One offer proposes to convert the Houses into apartments, another into townhouses, and a third into a sophisticated marketplace. Downtown has not yet seen the offer of a fourth developer, Absentia, Ltd. Absentia is unfamiliar with Downtown's other offers, but is confident that its offer will be appealing, although it is unwilling to reveal its exact plans. Each of the four offers presents a quite different financial package, each of which must be evaluated by Downtown in terms of present value. Both sides must take into consideration financial needs, tax implications, personal interests, and future dealings with the city Zoning Board. The negotiation involves attorneys representing Downtown Realty and Absentia, Ltd.